### PR TITLE
Generate set-unset golden output for gkehubfeaturemembership

### PR DESCRIPTION
**Pull request by ziyue-101**
### Change description

https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2027 got closed unexpectedly

Creating a new PR

### Tests you have done

- [ ] Run `make ready-pr` to ensure this PR is ready for review.
- [ ] Perform necessary E2E testing for changed resources.

### CODE REVIEW

**Comment by acpana** [comment]
Thanks for this 🙏🏼 

/lgtm
/approve

cc @maqiuyujoyce would you mind approving too?

This for us to capture the state currently w the TF controller. Once we cut over to the direct controller we may update these files.

File: tests/e2e/testdata/scenarios/fields/gkehubfeaturemembership/set_unset/script.yaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  projectRef:
+    external: projects/mock-project
+  location: global
+  membershipRef:
+    name: gkehubmembership-${uniqueId}
+  featureRef:
+    name: gkehubfeature-${uniqueId}
+  configmanagement:
+    configSync:
+        sourceFormat: unstructured
```
Comment (maqiuyujoyce) [inline tests/e2e/testdata/scenarios/fields/gkehubfeaturemembership/set_unset/script.yaml:67]: Could you update the indentation of this line?

**Review by maqiuyujoyce — COMMENTED** [review]
/lgtm
/approve

/hold

File: tests/e2e/testdata/scenarios/fields/gkehubfeaturemembership/set_unset/script.yaml (hunk 1 above)
Comment (ziyue-101) [reply tests/e2e/testdata/scenarios/fields/gkehubfeaturemembership/set_unset/script.yaml]: Done

**Review by maqiuyujoyce — COMMENTED** [review]
/hold cancel

**Pull request merged**